evap/staff/importers.py

```diff
@@ -92,6 +94,8 @@ def store_in_database(self, vote_start_datetime, vote_end_date, semester):
             semester=semester,
         )
         course.save()
```

Comment:
Why two times course.save()?

Reply:
the course instance must have an id before setting the many to many relation works.